rest/config_test.go

```diff
@@ -61,6 +64,14 @@ func TestReadServerConfig(t *testing.T) {
 			if test.err == "" {
 				assert.NoError(tt, err, "unexpected error for test config")
 			} else {
+				if test.errEE != "" &&
```

Comment:
If there's a case where test.err == "" and test.errEE isn't "", this check isn't going to happen.  If there isn't such a case (they should both be errors or empty), should we just skip the check on test.ErrEE being empty here since we've checked test.err above?

Reply:
Have renamed `test.err` variables to be more explicitly `stdlib`/`jsoniter`, and also reworked the assertion logic.